### PR TITLE
Finalization parameters added to the new grpc api

### PR DESCRIPTION
**Pull request by EmilLa1**
## Purpose

Introduce the new finalization committee parameters to the new grpc api. 

## Changes

Added the required types to the grpc api. 

- Added finalization committee parameters to the consensus v1 parameters
- Added to the update payload
- Added to pending chain updates
- Added to next sequence number for the specific new set of parameters. 
- (also added missing ones for the timeout parameters, min block time and max block energy which should have been introduced as part of https://github.com/Concordium/concordium-grpc-api/pull/33 )

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: v2/concordium/types.proto [hunk 1]
```diff
@@ -1323,6 +1323,19 @@ message TimeoutParameters {
   Ratio timeout_decrease = 3;
 }
 
+// Finalization committee parameters used from protocol version 6
+message FinalizationCommitteeParameters {
+  // The minimum number of bakers that can take part
+  // of the finalization committee.
```
Comment (abizjak) [inline v2/concordium/types.proto:1329]: This phrasing is confusing. It is odd to say "minimum" that "can".

If I read this I wonder whether this is a lower bound or something else. And it is not a lower bound in the sense that there can be fewer finalizers. Please elaborate.

File: v2/concordium/types.proto [hunk 2]
```diff
@@ -1323,6 +1323,19 @@ message TimeoutParameters {
...
+  uint32 minimum_finalizers = 1;
+  // The maximum number of bakers that can take part
+  // of the finalization committee.
```
Comment (abizjak) [inline v2/concordium/types.proto:1332]:
```suggestion
  // The maximum size of a finalization committee
```

File: v2/concordium/types.proto [hunk 3]
```diff
@@ -1323,6 +1323,19 @@ message TimeoutParameters {
...
+  uint32 maximum_finalizers = 2;
+  // The threshold for determining the stake required for being eligible the finalization committee.
+  // The amount is given by "total staked ccd / finalizer_relative_stake_threshold"
```
Comment (abizjak) [inline v2/concordium/types.proto:1335]: Use markdown please, so `total staked ccd / ...`

File: v2/concordium/types.proto (hunk 1 above)
Comment (EmilLa1) [reply v2/concordium/types.proto:1329]: // The minimum size of a finalization committee

**Review by td202 — APPROVED** [review]

**Review by abizjak — APPROVED** [review]

**Review by eb-concordium — APPROVED** [review]

**Pull request merged**